src/test/java/com/codeborne/selenide/SelectorsTest.java

```diff
@@ -57,35 +57,35 @@ void byTextEscapesQuotes() {
   void byAttributeUsesXPath() {
     By selector = Selectors.byAttribute("value", "катя");
     assertThat(selector)
-      .hasToString("By.cssSelector: [value='катя']");
+      .hasToString("By.cssSelector: [value=\"катя\"]");
   }
 
   @Test
   void byAttributeEscapesQuotes() {
     By selector = Selectors.byAttribute("value", "Ludvig'van\"Beethoven");
     assertThat(selector)
-      .hasToString("By.cssSelector: [value='Ludvig'van\"Beethoven']");
```

Comment:
Just a fun fact - this test case has produced the wrong locator all the way along, but test was passed :)